pkg/linear-pools/contracts/silo-linear-pool/SiloLinearPool.test.ts

```diff
@@ -193,17 +199,29 @@ describe('SiloLinearPool', function () {
   describe('getWrappedTokenRate', () => {
     context('under normal operation', () => {
       it('returns the expected value', async () => {
-        // Calculate the expected rate and compare to the getWrappedToken return value
-        const assetStorage = await mockSilo.assetStorage(mainToken.address);
-        // Get the 4th member from the struct 'total deposits'
-        const totalAmount = assetStorage[3];
+        const expectedRate = fp(2);
```

Comment:
For this basic version of the expected rate I think we can probably add a quick (single line) comment explaining how we got 2 (deposits / supply?)